packages/frontend-usagers/src/components/DS/informations-generales.vue

```diff
@@ -129,9 +131,14 @@ const saison = computed(() => {
 if (!operateurStore.operateurCourant) {
```

Comment:
on devrait ajouter un middleware qui fait cet appel à chaque fois sur la page.
Arrivé ici, on devrait être certain d'avoir l'opérateur. Si on ne l'a pas, on redirige ailleurs

Reply:
D'ailleurs, si le fetchMyOperateur plante, on va avoir plein d'effets de bord bizarre

Reply:
J'ai modifié le middleware has-id-demande : si pas d'operateurCourant ou s'il est not complet => on redirige vers le home.